dashboard/bin/sqs/process_queues_daemon

```diff
@@ -10,16 +10,26 @@ pwd  = File.dirname(File.expand_path(__FILE__))
 log_dir = Pathname.new(File.join(pwd, '../../log/')).cleanpath
 config_dir = Pathname.new(File.join(pwd, '../../config/')).cleanpath
 
-if ARGV.size < 1 || ARGV.size > 2
-  puts "Usage: #{$0} start|stop|restart|reload|status [CONFIG_FILE_PATH]"
-  exit
-end
-
-command = ARGV.shift
-if ARGV.size > 0
-  config_file = ARGV.shift
-else
-  config_file = "#{config_dir}/queue_config.json.erb"
+command = nil
+config_file = "#{config_dir}/queue_config.json.erb"
+
+OptionParser.new do |opts|
+  opts.on('-r', '--run COMMAND', 'The command to run - either start, stop, restart, reload, status') do |run_command|
```

Comment:
Tto work with minimal changes from the init script, it's important that this continue to allow an invocation of the from  "process_queues_daemon start", could you restore that option?